certbot/main.py

```diff
@@ -731,8 +731,9 @@ def register(config, unused_plugins):
     acc, acme = _determine_account(config)
     cb_client = client.Client(config, acc, None, None, acme=acme)
     # We rely on an exception to interrupt this process if it didn't work.
-    acc.regr = cb_client.acme.update_registration(acc.regr.update(
-        body=acc.regr.body.update(contact=('mailto:' + config.email,))))
+    acc_contacts = ['mailto:' + email for email in config.email.split(',')]
+    acc.regr.body.update(contact=acc_contacts)
```

Comment:
I think there's actually a bug here. These `acc.regr` and `acc.regr.body` objects are immutable and `update` returns a newly created object leaving the old object unmodified. We need to pass this new objects returned by `updated` into `update_registration`.

Reply:
If you want to take a shot at writing a unit test that would have caught this bug, it's probably a good idea, but you're welcome to skip this if you want. Doing so will require a fair bit of mocking. See `test_update_registration_with_email` in `certbot/tests/main_test.py` for an example.